src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -746,5 +751,49 @@ public static ScreenshotEvent onScreenshot(BufferedImage image, File screenshotF
         MinecraftForge.EVENT_BUS.post(event);
         return event;
     }
+    
+    public static List<List<BakedQuad>> sortQuadsByIndex(IBakedModel model)
+    {
+        List<List<BakedQuad>> layers = new ArrayList();
```

Comment:
Raw types -> generic types without their parameters. `ArrayList` here is raw, use `new ArrayList<List<BakedQuad>>()`